datafusion/sql/src/unparser/expr.rs

```diff
@@ -599,6 +611,19 @@ mod tests {
                 }),
                 "COUNT(DISTINCT *)",
             ),
+            (Expr::IsNotNull(Box::new(col("a"))), "\"a\" IS NOT NULL"),
+            (
+                Expr::IsTrue(Box::new((col("a") + col("b")).gt(lit(4)))),
+                "((\"a\" + \"b\") > 4) IS TRUE",
```

Comment:
What about using raw strings? https://doc.rust-lang.org/reference/tokens.html#raw-string-literals
It matches the other tests. :)

Reply:
Sure